src/components/Projects.tsx

```diff
@@ -28,216 +25,138 @@ import knife from "./assets/project-page/knife.png";
-          to=""
-          width={projectIconWidths}
-          position="absolute"
-          top={[200, 260, 260, 300, 300, 320, 440, 540, 360]}
-          left={[120, 120, 120, 160, 160, 400, 380, 560, 840]}
-        >
-          <img src={spider} alt="spider"></img>
-        </ProjectIconLink>
-
-        <ProjectIconLink
-          to=""
-          width={projectIconWidths}
-          position="absolute"
-          top={[300, 360, 380, 420, 420, 200, 380, 460, 220]}
-          left={[140, 140, 140, 160, 180, 780, 560, 760, 1040]}
-        >
-          <img src={magnify} alt="magnifying glass"></img>
-        </ProjectIconLink>
-      </ProjectIconContainer>
-
-      <BuyButton
-        position="absolute"
-        color="athensGray"
-        p={8}
-        py={1}
-        bg="transparent"
-        border={1}
-        borderColor="athensGray"
-        borderStyle="solid"
-        fontSize={[1, 2, 2, 3, 3, 3, 4, 5, 4]}
-        display="flex"
-        justifyContent="center"
-        right={[null, null, null, null, null, null, 40, 60, 60]}
-        bottom={[50, 50, 30, 60, 60, 60, 60, 60, 60]}
+        <Img
+          src={stairs}
+          gridColumn={[2, 2, 1]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center", "end"]}
+        />
+        <Img
+          src={shell}
+          gridColumn={2}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+          alignSelf={["initial", "initial", "center"]}
+        />
+        <Img
+          src={eye}
+          gridColumn={[2, 2, 3]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+        />
+        <Img
+          src={statue}
+          gridColumn={[2, 2, 1]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+        />
+        <Img
+          src={dragon}
+          gridColumn={2}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center", "start"]}
+          alignSelf={["initial", "initial", "end"]}
+        />
+        <Img
+          src={knife}
+          gridColumn={[2, 2, 3]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center", "start"]}
+        />
+        <Img
+          src={mask}
+          gridColumn={[2, 2, 1]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+        />
+        <Img
+          src={spider}
+          gridColumn={[2, 2, 2]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+          alignSelf={["initial", "initial", "center"]}
+        />
+        <Img
+          src={magnify}
+          gridColumn={[2, 2, 3]}
+          maxWidth={["75%", "75%", "50%", "30%"]}
+          justifySelf={["center", "center"]}
+        />
+      </Container>
+      <BuyButtonContainer position="fixed" top="50%" zIndex={1}>
```

Comment:
I was going to say I´d probably add some props here for desktop so that the buy button shows in the bottom left but on second thoughts I think it looks quite good like this so let´s leave it